### PR TITLE
Remove Extra approvals from zap stake and zap bond

### PR DESCRIPTION
**Pull request by calusaca**

### CODE REVIEW

File: contracts/zaps/OlympusZapV2.sol [hunk 1]
```diff
@@ -116,7 +116,7 @@ contract Olympus_V2_Zap_In is ZapBaseV3 {
         uint256 toInvest = _pullTokens(fromToken, amountIn, referral, true);
 
         // make sure "swapTarget" is approved to spend this contracts "fromToken"
-        _approveToken(fromToken, swapTarget, toInvest);
+        //_approveToken(fromToken, swapTarget, toInvest);
```
Comment (suhailgme) [inline contracts/zaps/OlympusZapV2.sol]: You can remove this line instead of commenting it out.

File: contracts/zaps/OlympusZapV2.sol [hunk 2]
```diff
@@ -78,7 +78,7 @@ contract Olympus_V2_Zap_In is ZapBaseV3 {
         uint256 toInvest = _pullTokens(fromToken, amountIn, referral, true);
 
         // approve "swapTarget" to spend this contracts "fromToken" if needed
-        _approveToken(fromToken, swapTarget, toInvest);
+        //_approveToken(fromToken, swapTarget, toInvest);
```
Comment (suhailgme) [inline contracts/zaps/OlympusZapV2.sol]: You can remove this line instead of commenting it out.

File: contracts/zaps/OlympusZapV2.sol [hunk 3]
```diff
@@ -78,7 +78,7 @@ contract Olympus_V2_Zap_In is ZapBaseV3 {
         uint256 toInvest = _pullTokens(fromToken, amountIn, referral, true);
 
         // approve "swapTarget" to spend this contracts "fromToken" if needed
```
Comment (suhailgme) [inline contracts/zaps/OlympusZapV2.sol]: These comments can be removed as well

**Review by suhailgme — CHANGES_REQUESTED** [review]
You can just remove the lines instead of commenting them out.

File: contracts/zaps/OlympusZapV2.sol (hunk 2 above)
Comment (calusaca) [reply contracts/zaps/OlympusZapV2.sol]: Comments removed

File: contracts/zaps/OlympusZapV2.sol (hunk 3 above)
Comment (calusaca) [reply contracts/zaps/OlympusZapV2.sol]: Comments removed

File: contracts/zaps/OlympusZapV2.sol (hunk 1 above)
Comment (calusaca) [reply contracts/zaps/OlympusZapV2.sol]: Comments removed

File: contracts/zaps/OlympusZapV2.sol (hunk 2 above)
Comment (calusaca) [reply contracts/zaps/OlympusZapV2.sol]: Comments removed

**Review by suhailgme — APPROVED** [review]

**Pull request merged**